licenses/bin/LatencyUtils-2.0.3.txt

```diff
@@ -0,0 +1,27 @@
+BSD License
+
+Copyright (c) 2000-2015 www.hamcrest.org
```

Comment:
Are you sure LatencyUtils is copyrighted by hamcrest?

Reply:
agreed